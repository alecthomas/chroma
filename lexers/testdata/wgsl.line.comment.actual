  // this is a line-ending comment
 //* embed a bock comment start, after a space
// /* embed a bock comment start, v2
