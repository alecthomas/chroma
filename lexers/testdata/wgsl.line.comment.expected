[
  {"type":"TextWhitespace","value":"  "},
  {"type":"CommentSingle","value":"// this is a line-ending comment\n"},
  {"type":"TextWhitespace","value":" "},
  {"type":"CommentSingle","value":"//* embed a bock comment start, after a space\n// /* embed a bock comment start, v2\n"}
]
